src/components/Checkbox.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <label :for="id" class="checkbox-label" :class="labelClasses">
+    <input
+      :id="id"
+      type="checkbox"
+      class="checkbox"
+      :checked="checked"
+      :name="inputName"
+      :value="inputValue"
+      :disabled="disabled"
+      @change="onChange"
+    /><span>
```

Comment:
Is the `span` wrapper necessary for the label text? I removed it locally and it didn't seem to effect the visual output at all.

Reply:
It did change the visual output for me in the beginning, but that might be because I hadn't deleted all the sass styles before, I will check tomorrow.